climakitae/util/utils.py

```diff
@@ -520,3 +521,42 @@ def summary_table(data):
         df = df.sort_values(by=["year"])
 
     return df
+
+
+def convert_to_local_time(data, selections, lat, lng) -> xr.Dataset:
+    """
+    Converts the inputted data to the local time of the selection.
+    """
+    # 1. Find the other data
+    start, end = selections.time_slice
+    tz_selections = copy.copy(selections)
+    tz_selections.time_slice = (
+        end + 1,
+        end + 1,
+    )  # This is assuming selections passed with be negative UTC time. Also to get the next year of data.
+
+    tz_data = tz_selections.retrieve()
+
+    # 2. Combine the data
+    total_data = xr.concat([data, tz_data], dim="time")
+
+    # 3. Change datetime objects to local time
+    tf = TimezoneFinder()
+    local_tz = tf.timezone_at(lng=lng, lat=lat)
+    new_time = (
+        pd.DatetimeIndex(total_data.time)
+        .tz_localize("UTC")
+        .tz_convert(local_tz)
+        .tz_localize(None)
+        .astype("datetime64[ns]")
+    )
+    total_data["time"] = new_time
+
+    # 4. Subset the data by the initial time
+    start = data.time[0]
+    end = data.time[-1]
+    sliced_data = total_data.sel(time=slice(start, end))
+
+    print("Data converted to local timezone!")
```

Comment:
```suggestion
    print("Data converted to {}".format(local_tz)
```

Could be nice to confirm which timezone was converted to!